js/main.js

```diff
@@ -46,7 +46,7 @@ if (shouldQuit) {
 
 var symphonyAutoLauncher = new AutoLaunch({
     name: 'Symphony',
-    path: process.execPath,
+    path: isMac ? '/Applications/' + app.getName() + '.app' : process.execPath,
```

Comment:
what about this..
var appPath = app.getPath('exe').split('.app/Content')[0] + '.app';
appLauncher = new AutoLaunch({
  name: app.getName(),
  path: appPath,
});